### DISCUSSION TITLE
Tailwind Plugin Mantine

### DISCUSSION THREAD

**Discussion by Sajarin-M** [mantinedev/mantine#6453] [Show]
🚀 Check out this tailwind plugin designed to sync your Mantine theme values with Tailwind CSS. 
For example, `theme.radius.md` can be available in Tailwind as `rounded-md`.

✨ Please take a look and share your feedback. Thank you! 🙏

[Github](https://github.com/Sajarin-M/tailwind-plugin-mantine)
[NPM](https://www.npmjs.com/package/@devoss/tailwind-plugin-mantine)

**Comment by bsolca** [comment]
I would love to have a project example showcase. I was not able to make it work @Sajarin-M .

**Reply by Sajarin-M** [reply]
Here is an example of a [Next.js project](https://github.com/Sajarin-M/tailwind-plugin-mantine-example), and here is the same on [CodeSandbox](https://codesandbox.io/p/devbox/gmmnv8). Please check
